src/test/munit/success-requests/success-requests-test.xml

```diff
@@ -234,6 +234,28 @@
     </munit:validation>
   </munit:test>
 
+  <munit:test name="raw-request-uri">
+    <munit:enable-flow-sources>
+      <munit:enable-flow-source value="success-server-main"/>
+      <munit:enable-flow-source value="get:\orders:success-server-config"/>
+    </munit:enable-flow-sources>
+    <munit:execution>
+      <http:request method="GET" config-ref="http-requester-simple" path="api/odata.svc/orders?$filter=CustomerID eq '12%25CD'">
+        <http:headers>#[output application/java --- {"Accept" : "application/json"}]</http:headers>
+      </http:request>
+    </munit:execution>
+    <munit:validation>
+      <munit-tools:assert-that expression="#[output application/java --- attributes.statusCode]" is="#[MunitTools::equalTo(200)]"/>
+      <munit-tools:retrieve key="attributes"/>
+      <munit-tools:assert-that expression="#[output application/java --- payload.rawRequestUri ]"
```

Comment:
why not strict assertions ? instead of contains

Reply:
shortcut to avoid escaping, good catch